c/ckb_syscalls.h

```diff
@@ -62,6 +62,10 @@ int ckb_load_witness(void* addr, volatile uint64_t* len, size_t offset,
   return syscall(SYS_ckb_load_witness, addr, len, offset, index, source, 0);
 }
 
+int ckb_load_args(void* addr, volatile uint64_t* len, size_t offset) {
```

Comment:
Instead of a syscall only capable of loading args, can we build a syscall that loads the current running script object instead? The reason for this is:

1. In practice, there will be requirement for loading the `code_hash` part of current running script
2. Due to the way serialization works now, users can still only load the args part as they wish via a special offset, so current use case won't be affected

Reply:
Updated, please review again, thanks